### DISCUSSION TITLE
this is the gorbinos

### DISCUSSION THREAD

**Discussion by jaiguy999** [aeolus-1/triangleGame.github.io#38] [General]
"Apply pressure to the W key to jump",
"This is the Gorbnos quest of github games",
"You can vote for color themes on the github",
"This is not really loading it all fake",
"If you really know me, you will know my Father as well. From now on, you do know him and have seen him. John 14:7",
"When anxiety was great within me, your consolation brought me joy. Psalm 94:19",
"I consider that our present sufferings are not worth comparing with the glory that will be revealed in us. Romans 8:18",
"To this you were called, because Christ suffered for you, leaving you an example, that you should follow in his steps. 1 Peter 2:21",
"Read this the end. I don't normally tell people this but you are the one exception. Im usally really untrusting at least thats what my wife says. Anyways, the secret to"
“if you want to win then don’t fall :)”,
“If you when you when the you are the when you are”,
“Have you tried to git gud?”,
“Do you really wanna? Do you really wanna taste it?”,
“Look at you with your dry ass lips”,
“To show you the power of flex tape”,
“I sawed this tip in half”,
“Jai wuz here”,
“Sasha will you go out with me?”,
“Patty smells like poo”,
“Emma please come back”,
“Can anyone drop some xan or md in Samford”,
“Please help me ben has locked me in a room to write tips for his game”,
“There is a dead body at the bush behind d block”,
“Do it jiggle?”,
“There are 12 versions of the show ‘the office’  such as a Indian, German and Finnish versions”,
“Australia lost a war to birds”,
“lasnga -garfield”, 
“Make sure a teacher isn’t behind you”,
“Send nudes”,
“I used to play triangle game like you until I took an arrow to the knee”,
“Mr Behan is behind you”,
“I don’t support Ukraine”,
“Uhhhh mr witter were out of meth uhhh jesser”,
“Ok stop playing this is gonna be on the test”,
“Its called triangle game cause there’s on at the end”,
“There’s a red spy in the base!”,
“Uhh moty I turned my self into a cucumber, I’m cucumber rig”,
“Hey look at the girls ass”,
“If you see bugs or want to suggest new things to be added you can do so at the GitHub page for the game”,
“This game has vr support, don’t believe me? Shove your finger up your ass”,
“Hey now your a porn star, get you clothes off, get laid”,
“STANDING HERE, I REALISE, YOU WERE JUST LIKE ME”,
“So I walking down the street the other day when I saw this sign that said wood fired pizza, wood fired pizza I said? How is pizza gonna get a job now?”,
“Not made using unitys particle system”,
“Fortnite battle pass I just shit out my ass”,
“Family guy funny moments”,
“Welp off to go plow my mistress”,
“Raiden, Raiden! RAIIIIDEEEEN”,
“Type ‘joinpl’ while in game”,
“Triangle game and knuckles”,
“I should dip my balls in sulphuric acid”,
“Eat my shorts”,
“Wealth, fame, power, gold Rodger the king of the pirates had obtained this and everything else the world had to offer”,
“I make a da shit in da terlet”,
“My balls a stereo”,
“If you read this ur gay”,
“This is the gorbinos quest of GitHub games”,

I am extra dumb dumb

**Comment by nekoify** [comment]
fortnite battle pass
